test/testobjects/TestOrchestrations.ts

```diff
@@ -1,7 +1,8 @@
 import * as df from "../../src";
 
 export class TestOrchestrations {
-    public static NotGenerator: any = df.orchestrator(function* (context: any) {
+    public static NotGenerator: any = df.orchestrator(function* (_context: any) {
+        void _context;

```

Comment:
why do we need this `_context` variable}

Reply:
I thought it would throw a typescript error if I tried to remove it because then the function no longer conforms to the type expected by `df.orchestrator`, but turns out I was wrong so I removed it